clients/drcachesim/CMakeLists.txt

```diff
@@ -41,6 +41,25 @@ else ()
   set(os_name "unix")
```

Comment:
> Since the stability issues with drbbdup on larger apps seem limited to x86, we enable -align_endpoints by default for AArchXX.

So did we do more tests on AArch64 with `-align_endpoints` actually on? And found that it is indeed more stable on AArch64 than X86?

Reply:
Yes, a (proprietary) detach stress test has a 100% pass rate on arm with this option on, yet fails consistently ~2% of the time on x86.  A larger (proprietary) app has passed a bunch of times in a row with this option on arm where on x86 it fails about half the time.  I am now focusing on x86-specific aspects of bbdup to try to figure out these stability problems -- haven't found the issue yet.